runtime/dotnet/core/ComposerBotPathAdapter.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+
+namespace Microsoft.BotFramework.Composer.Core
+{
+    /// <summary>
+    /// Bot path adapter, for development environment, use '../../' as the bot path, for deployment and production environment, use 'ComposerDialogs' as bot path
+    /// </summary>
+    public static class ComposerBotPathAdapter
```

Comment:
Naming: Should this be called `ComposerBotPathExtensions` or something that ends with `*Extensions`? When we think of Adapter I tend to think of the Adapter object oriented pattern but instead here we have extension methods.